### PR TITLE
Update Fatih Kalifa's profile

### PR DESCRIPTION
**Pull request by asendia**
## Summary

This commit sole purpose is to appreciate technical excellence of Fatih Kalifa. 

## Test Plan

1) Secretly create this PR
2) Waiting for the vote result
3) Bump old post in internal slack channel regarding this update 😬

## Checklist

- [x] This PR contains relevant automated test(s) with my proposed changes
- [x] I have tested this changes in at least 2 browsers (if relevant)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by fawwaz — APPROVED** [review]
~LGTM~ Commented

**Review by masbagal — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -133,13 +133,13 @@
   - Medium: [@108kb](https://medium.com/@108kb)
   
 + **Fatih Kalifa**
-  - Skills: #React, #NodeJs
+  - Skills: #React, #NodeJs, #Svelte, #Rust, #GraphQL
```
Comment (fawwaz) [inline README.md]: #ReasonML

**Review by fawwaz — CHANGES_REQUESTED** [review]
Please add ReasonML.

File: README.md (hunk 1 above)
Comment (asendia) [reply README.md]: Added

**Review by fawwaz — APPROVED** [review]

**Pull request merged**